src/components/button.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+Create a button that has 2 variants (primary, secondary) and 2 sizes (default, small)
+A button can be also disabled
+The button can take text to display
+*/
+
+import { cn } from "../utils/styling";
+
+type Props = {
+  children: React.ReactNode;
+  variant?: "primary" | "secondary";
+  size?: "default" | "small";
+};
+
+export function Button({
+  children,
+  variant = "primary",
+  size = "default",
+}: Props) {
+  return (
+    <button
+      className={cn(
+        "rounded-lg font-semibold",
+        "hover:shadow-xl",
```

Comment:
we don't show a shadow on hover

Reply:
Ja, das und Ghost hatte ich auch rausoperiert und es war dann doch wieder drin. :O